app/static/js/emailModal.js

```diff
@@ -16,11 +16,17 @@ function retrieveEmailTemplateData() {
   });
 }
 
-function showEmailModal(eventID, programID, selectedTerm) {
+function showEmailModal(eventID, programID, selectedTerm, isPastEvent) {
   $(".modal-body #eventID").val(eventID);
   $(".modal-body #programID").val(programID);
   $(".modal-body #selectedTerm").val(selectedTerm);
 
+  if (isPastEvent) {
+    $(".pastEventWarning").prop("hidden", false);
+  } else {
+    $(".pastEventWarning").prop("hidden", true);
+  }

```

Comment:
I would probably just use the boolean value of isPastEvent in the prop() call.
```
var hidePastEventWarning = !isPastEvent;
$(".pastEventWarning").prop("hidden", hidePastEventWarning);
```
or a one-liner: `$(".pastEventWarning").prop("hidden", !isPastEvent);`

Probably more of a personal style thing, since the negatives can affect readability.